src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,21 @@
 package mate.academy;
 
+import mate.academy.lib.Injector;
+import mate.academy.model.Movie;
+import mate.academy.service.MovieService;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+    private static final MovieService movieService =
+            (MovieService) injector.getInstance(MovieService.class);
 
+    public static void main(String[] args) {
+        Movie terminator = new Movie("terminator", "Awesome terminator!");
+        Movie shrek = new Movie("Shrek 1", "Shrek is love!");
+        System.out.printf("...Insert movies into DB...\n");
```

Comment:
why do use prinitf if you are not using it functional? It is better to use println instead.